core/templates/dev/head/components/summary_tile/CollectionSummaryTileDirective.js

```diff
@@ -29,28 +31,37 @@ oppia.directive('collectionSummaryTile', [function() {
       getNodeCount: '&nodeCount',
       getLastUpdatedMsec: '&lastUpdatedMsec',
       getThumbnailIconUrl: '&thumbnailIconUrl',
-      getThumbnailBgColor: '&thumbnailBgColor'
+      getThumbnailBgColor: '&thumbnailBgColor',
+      isLinkedToEditorPage: '=?isLinkedToEditorPage'
     },
     templateUrl: 'summaryTile/collection',
     controller: [
       '$scope', 'oppiaDatetimeFormatter', 'UrlInterpolationService',
-      'COLLECTION_VIEWER_URL', function($scope, oppiaDatetimeFormatter,
-      UrlInterpolationService, COLLECTION_VIEWER_URL) {
+      'COLLECTION_VIEWER_URL', 'COLLECTION_EDITOR_URL', function($scope,
+      oppiaDatetimeFormatter, UrlInterpolationService, COLLECTION_VIEWER_URL,
+      COLLECTION_EDITOR_URL) {
+        $scope.DEFAULT_EMPTY_TITLE = 'Untitled';
         $scope.getLastUpdatedDatetime = function() {
           return oppiaDatetimeFormatter.getLocaleAbbreviatedDatetimeString(
             $scope.getLastUpdatedMsec());
         };
 
-        $scope.wasRecentlyUpdated = function() {
-          return oppiaDatetimeFormatter.isRecent($scope.getLastUpdatedMsec());
-        };
-
         $scope.getCollectionLink = function() {
-          return UrlInterpolationService.interpolateUrl(
-            COLLECTION_VIEWER_URL, {
-              collection_id: $scope.getCollectionId()
+          if (angular.isDefined($scope.isLinkedToEditorPage)) {
+            if ($scope.isLinkedToEditorPage) {
```

Comment:
Actually there's a bit too much duplicated code here. Define a targetUrl variable and do:

    var targetUrl = (
      $scope.isLinkedToEditorPage ? ... : ...);

Then remove the if/else.